p2p/protocol/identify/id_test.go

```diff
@@ -80,7 +80,7 @@ func subtestIDService(t *testing.T) {
 	// the IDService should be opened automatically, by the network.
 	// what we should see now is that both peers know about each others listen addresses.
 	t.Log("test peer1 has peer2 addrs correctly")
-	testKnowsAddrs(t, h1, h2p, h2.Peerstore().Addrs(h2p))        // has them
+	testKnowsAddrs(t, h1, h2p, h2.Addrs())                       // has them
 	testHasCertifiedAddrs(t, h1, h2p, h2.Peerstore().Addrs(h2p)) // should have signed addrs also
```

Comment:
I'm not sure I understand all the test changes. Why are we sometimes using `h.Addrs()` and sometimes using `h.Peerstore().Addrs(h.ID)`?